packages/blueprints-v2-addon/src/blueprints/run-new/__addonLocation__/tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "allowJs": true,
     "declarationDir": "declarations",
     "rootDir": "./src",
-    "skipLibCheck": true,
```

Comment:
Already set to `true` by [`@tsconfig/ember@3.0.10`](https://www.npmjs.com/package/@tsconfig/ember/v/3.0.10?activeTab=code).